src/config_ctrl.js

```diff
@@ -8,10 +8,26 @@ export class OCIConfigCtrl {
   /** @ngInject */
   constructor ($scope, backendSrv) {
     this.backendSrv = backendSrv
-    this.tenancyOCID = this.current.jsonData.tenancyOCID
-    this.defaultRegion = this.current.jsonData.defaultRegion
     this.environment = this.current.jsonData.environment      
     this.tenancymode = this.current.jsonData.tenancymode
+    this.defaultRegion = this.current.jsonData.defaultRegion
+    this.region0 = this.current.jsonData.region0
+    this.region1 = this.current.jsonData.region1
+    this.region2 = this.current.jsonData.region2
+    this.region3 = this.current.jsonData.region3
+    this.region4 = this.current.jsonData.region4
+    this.region5 = this.current.jsonData.region5
+    this.profile0 = this.current.jsonData.profile0
+    this.profile1 = this.current.jsonData.profile1
+    this.profile2 = this.current.jsonData.profile2
+    this.profile3 = this.current.jsonData.profile3
+    this.profile4 = this.current.jsonData.profile4
+    this.profile5 = this.current.jsonData.profile5
+    this.addon1 = this.current.jsonData.addon1
+    this.addon2 = this.current.jsonData.addon2
+    this.addon3 = this.current.jsonData.addon3
+    this.addon4 = this.current.jsonData.addon4
+
```

Comment:
What is the plan for error handling in single and multiple tenancy configuration data? I realize that "Save and Test" does not work currently but when fixed will it tell me which of the 5 tenancy is bad??

Reply:
Similar question on individual field validation. For example what happens if I   put the same profile name to all 5??